includes/classes/Blocks/SimplifiedSummaryBlock.php

```
@@ -0,0 +1,125 @@
+	 * Loop) and falls back to the global post. Renders regardless of the
+	 * edac_simplified_summary_prompt option because manual placement is a
+	 * deliberate act, matching edac_get_simplified_summary().
+	 *
+	 * @since 1.xx.x
+	 *
+	 * @param array    $attributes The block attributes.
+	 * @param string   $content    The block content.
+	 * @param WP_Block $block      The block instance.
+	 * @return string
+	 */
+	public function render( $attributes, $content, $block ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)